ekf_general/DataSets/.svn/all-wcprops

```diff
@@ -0,0 +1,41 @@
+K 25
```

Comment:
@ignaciotb Did you mean to check in all these files in the .svn directory?

Reply:
nop! The folder was hidden